src/components/MapArea.js

```diff
@@ -64,20 +98,19 @@ export default function MapArea() {
   });
 
   const changeStyle = (styleURL) => {
-    console.log('styleURL -> ', styleURL)
     setStyle(styleURL);
   }
 
 	return (
     <StyledMapAreaContainer>
-      <StyleBar class="mapStyleBar" onChange={e => changeStyle(e.target.value)}>
-        <input type="radio" id="streets-v11" value="mapbox://styles/mapbox/streets-v11" name="Style"/>
+      <StyleBar className="mapStyleBar" onChange={e => changeStyle(e.target.value)}>
+        <input type="radio" id="streets-v11" value="mapbox://styles/mapbox/streets-v11" name="Style" defaultChecked={style === "mapbox://styles/mapbox/streets-v11"}/>
         <label htmlFor="streets-v11">Streets</label>
-        <input type="radio" id="outdoors-v11" value="mapbox://styles/mapbox/outdoors-v11" name="Style"/>
+        <input type="radio" id="outdoors-v11" value="mapbox://styles/mapbox/outdoors-v11" name="Style" defaultChecked={style === "mapbox://styles/mapbox/outdoors-v11" }/>
         <label htmlFor="outdoors-v11">Outdoors</label>
-        <input type="radio" id="light-v10" value="mapbox://styles/mapbox/light-v10" name="Style"/>
+        <input type="radio" id="light-v10" value="mapbox://styles/mapbox/light-v10" name="Style" defaultChecked={style === "mapbox://styles/mapbox/light-v10"}/>
         <label htmlFor="light-v10">Light</label>
-        <input type="radio" id="dark-v10" value="mapbox://styles/mapbox/dark-v10" name="Style"/>
+        <input type="radio" id="dark-v10" value="mapbox://styles/mapbox/dark-v10" name="Style" defaultChecked={style === "mapbox://styles/mapbox/dark-v10"}/>
         <label htmlFor="dark-v10">Dark</label>
```

Comment:
these labels and inputs have a lot of common code, let's dry it up by creating a base layers array, and looping through that here to display our inputs/labels